scripts/devenv-builder/configure-vm.sh

```diff
@@ -88,29 +86,41 @@ if ${INSTALL_BUILD_DEPS} || ${BUILD_AND_RUN}; then
     sudo systemctl enable --now cockpit.socket
 fi
 
+GO_VER=1.20.4
```

Comment:
Given change in `microshift.spec`, should we remove `golang` from `dnf install`?